### DISCUSSION TITLE
88A0BA1C-2363-4FDC-BB97-970197D3C552

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#56] [General]
# 88A0BA1C-2363-4FDC-BB97-970197D3C552

HD 141399 is a K-type main sequence star about 120.86 light-years away in the constellation Bootes. It has a total of four gas giant planets, all of which are larger in radii than Jupiter itself

https://celestia.mobi/addon?item=88A0BA1C-2363-4FDC-BB97-970197D3C552&lang=zh_CN

**Comment by 13940951572** [comment]
KD131399AB没有？
